index.html

```diff
@@ -106,7 +106,7 @@
         <div class="buttons">
           <a
             class="toggle-theme"
-            href="#toggleTheme"
+            href="#"
```

Comment:
Can you change it back? Now when clicking on the button it takes you to the top of the page

Reply:
ok. I'll add it back.